config.md

```diff
@@ -608,6 +630,29 @@ Here is a full example `config.json` for reference.
         ],
         "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
     },
+    "solaris": {
```

Comment:
We need a better way to handle this, because putting a `solaris` section in a config that has `linux` in `platform.os` may be confusing.